.github/workflows/elixir-ci.yml

```diff
@@ -58,7 +58,9 @@ jobs:
       run: mix dialyzer
 
     - name: Execute tests
-      run: mix coveralls.json --warnings-as-errors
+      run: |
+        mix test --only integration --warnings-as-errors --max-cases 1 --cover --export-coverage integration-coverage

```

Comment:
`--max-cases 1` has no effect here. ExUnit will run `async: true` modules in parallel with each other, but runs the tests within each module in series. Since there is only one module (aka case) with the `integration` tag, there will only be one case being run no matter what.